app/views/partials/swagger/deprecated/tags.hbs

```diff
@@ -20,7 +21,7 @@
       <tbody>
         {{#each operations}}
         <tr>
-          <td><a href="#operation-{{htmlId path}}-{{htmlId method}}">{{toUpperCase method}} {{path}}</a></td>
+          <td><a href="#operation-{{htmlId path}}-{{htmlId tags.[0]}}">{{toUpperCase method}} {{path}}</a></td>
```

Comment:
Hopefully it's safe to assume that index is always there? But maybe handlebars fails silently when that happens?

Reply:
It won't break at the non-existent index, but would break in the `htmlId` function call when it tries to use an `undefined` value. However, it's impossible for an `operation` not to have a tag. They're built up in a way where they will have exactly 1 tag.